src/platform/common/constants.ts

```diff
@@ -43,22 +43,6 @@ export const DEFAULT_INTERPRETER_SETTING = 'python';
 
 export const STANDARD_OUTPUT_CHANNEL = 'STANDARD_OUTPUT_CHANNEL';
 
-export const isCI = process.env.TF_BUILD !== undefined || process.env.GITHUB_ACTIONS === 'true';
```

Comment:
Cant use process in web. Well not without predefining every variable we might want and that's not really the intent. 

Most of these I changed to global state that is set by other locations.

Reply:
> Well not without predefining every variable we might want and that's not really the intent.

Aah got it, I think you've answered the question i had.